apps/web/src/presentation/react/components/NetworkSwitch.tsx

```diff
@@ -0,0 +1,111 @@
+import { ComponentProps, MutableRefObject, ReactNode, useState } from 'react'
+import { Menu } from './Menu'
+import { useOnClickOutside } from '../hooks/useOnClickOutside'
+import { CheckMarkIcon } from './icons/CheckMark'
+import { Chevron } from './icons/Chevron'
+import React from 'react'
+
+type Props = {
+  className?: string
+  networks: string[]
+  selectedNetwork: string
+  selectNetwork: (value: string) => void
+}
+
+const MenuContainer = ({
+  children,
+  refProp,
+  className,
+  ...props
+}: { children: ReactNode } & {
+  refProp: React.RefObject<HTMLDivElement> | MutableRefObject<HTMLDivElement>;
+} & ComponentProps<'div'>) => (
```

Comment:
`children` _should_ be already inluded in `ComponentProps<'div'>`

```suggestion
}: {
  refProp: React.RefObject<HTMLDivElement> | MutableRefObject<HTMLDivElement>;
} & ComponentProps<'div'>) => (
```

Reply:
Done.